add `command list --device <devicename>

difince opened the pull request:
add `command list --device <devicename>
--device is mandaroty field

Signed-off-by: Diana Atanasova <dianaa@vmware.com>

mkbhanda commented on cmd/command/list/list.go:
```diff
@@ -0,0 +1,70 @@
+package list
+
+import (
+	"github.com/spf13/viper"
```
should this import be at the very end?

difince replied:
It should be :) Thank you

mkbhanda commented on cmd/command/list/list.go:
```diff
@@ -0,0 +1,70 @@
+	}
+	pw := viper.Get("writer").(io.WriteCloser)
+	w := new(tabwriter.Writer)
+	w.Init(pw, 0, 8, 1, '\t', 0)
```
Would you add a comment to explain what all these magic numbers are 0, 8, 1

difince replied:
These numbers are described in `w.Init`method itself.

> // A Writer must be initialized with a call to Init. The first parameter (output)
// specifies the filter output. The remaining parameters control the formatting:
//
//	**minwidth**	minimal cell width including any padding
//	**tabwidth**	width of tab characters (equivalent number of spaces)
//	**padding**		padding added to a cell before computing its width
//	**padchar**		ASCII char used for padding
//			        if padchar == '\t', the Writer will assume that the
//	         		width of a '\t' in the formatted output is tabwidth,
//		        	and cells are left-aligned independent of align_left
//			       (for correct-looking results, tabwidth must correspond
//			       to the tab width in the viewer displaying the result)
//	**flags**		formatting control
//
func (b *Writer) Init(output io.Writer, minwidth, tabwidth, padding int, padchar byte, flags uint) *Writer {

So, I think there is not need to duplicate this information in all the places that uses `Writer.Init` method.

difince replied:
In follow-up PR, I plan to extract the logic with these magical numbers in a common place used from all the commands, so this place could be appropriate for adding this clarification.

difince commented:
@mkbhanda I have added support to get All  Commands , but without --limit flag. This is because `edgex-go` does not support the functionality you have requested - " 50 internal maximum". Probably, this list could get pretty exhausting, but this endpoint still exists in edgex-go, so I have added it.
